### PR TITLE
Adding additional field options.

### PR DESCRIPTION
**Pull request by jhosmer**
Add fields grpThreads sampleCount errorCount responseHeaderSize responseSize URL to FFW.

### CODE REVIEW

File: standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java [hunk 1]
```diff
@@ -44,7 +44,9 @@
             + "startTimeMillis endTimeMillis "
             + "responseTimeMicros latencyMicros "
             + "requestData responseData responseHeaders "
-            + "threadsCount requestHeaders connectTime";
+            + "threadsCount requestHeaders connectTime "
+            + "grpThreads SampleCount ErrorCount "
```
Comment (undera) [inline standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java]: please use same style like other fields 
SampleCount=>sampleCount, ErrorCount=>errorCount

**Comment by undera** [comment]
Thanks for contribution. According to http://jmeter-plugins.org/wiki/DeveloperGuide/ you need to change the doc page for FFW and also add a record to changelog about your improvement

**Comment by jhosmer** [comment]
Wiki doc page updated (and cleaned up a little) for FFW.  Added note to Changelog.

**Comment by undera** [comment]
The wiki page does not use GitHub markdown, it uses old Google Code markup syntax. That's why the file extensions are .wiki and not .md. Please use old formatting for the table. Otherwise it will become broken on the website.

File: standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java [hunk 2]
```diff
@@ -44,7 +44,9 @@
             + "startTimeMillis endTimeMillis "
             + "responseTimeMicros latencyMicros "
             + "requestData responseData responseHeaders "
-            + "threadsCount requestHeaders connectTime";
+            + "threadsCount requestHeaders connectTime "
```
Comment (undera) [inline standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java]: Here I see "threadsCount" while other places show "allThreads" isn't this a mistake?

Comment (jhosmer) [inline standard/src/kg/apc/jmeter/reporters/FlexibleFileWriter.java]: The threadsCount field existed but there was no button for it in the GUI.  When adding the button label I used "allThreads" to be consistent with other listeners/plugins but I'll change it to match the internal field name of "threadsCount".

**Comment by undera** [comment]
Thanks for contributing!

**Pull request merged**